src/Orleans/GrainDirectory/MultiClusterStatus.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.GrainDirectory
+{
+    /// <summary>
+    /// Status of a directory entry with respect to multi-cluster registration
+    /// </summary>
+    internal enum MultiClusterStatus
```

Comment:
Should we also have a `None` value for when we don't use Multi-Cluster or is `Owned` the correct logical value to use in this case?